### PR TITLE
More css fixes

### PR DESCRIPTION
**Pull request by Gwenwyfar**

### CODE REVIEW

**Review by XinYenFon — APPROVED** [review]

File: Themes/default/css/index.css [hunk 1]
```diff
@@ -3522,6 +3487,11 @@ h3.catbg .generic_icons::before, h3.catbg .icon {
 	vertical-align: middle;
 	margin: 0 5px 0 0;
 }
+.cat_bar + .windowbg2, .cat_bar + .windowbg, .cat_bar + .roundframe {
```
Comment (XinYenFon) [inline Themes/default/css/index.css]: I'm just curious related to this and missing .information + .windowbg/2 rest is nice ^^

**Review by XinYenFon — APPROVED** [review]

File: Themes/default/Calendar.template.php [hunk 2]
```diff
@@ -1025,13 +1025,13 @@ function template_hms()
...
 			<tr>
-				<th class="windowbg2" style="font-weight: bold; text-align: center; border-bottom: 1px solid #ccc;">Binary Clock</th>
+				<th class="windowbg" style="font-weight: bold; text-align: center; border-bottom: 1px solid #ccc;">Binary Clock</th>
 			</tr>';
 
 	foreach ($context['clockicons'] as $t => $v)
 	{
 		echo '
-			<tr class="', $alt ? 'windowbg2' : 'windowbg', '">
+			<tr class="', $alt ? 'windowbg' : 'windowbg', '">
```
Comment (XinYenFon) [inline Themes/default/Calendar.template.php]: :P

**Review by XinYenFon — CHANGES_REQUESTED** [review]
You also need to clean up the $alt definitions from removed functions.

**Comment by Gwenwyfar** [comment]
You're aware that entire section is the clock easter egg right?

**Comment by XinYenFon** [comment]
Not really :P I just checked the code 😆 not the func.

**Comment by jdarwood007** [comment]
Quick testing shows nothing I see as breaking.  Antes gave a thumbs up as well

**Pull request merged**